### DISCUSSION TITLE
Updating vendor/cache with bundler

### DISCUSSION THREAD

**Discussion by jdelStrother** [renovatebot/renovate#10990] [Request Help] [answered]
Not sure if this is a feature request, or if I'm just missing the configuration option - is it possible to get Bundler updates to include the cached gems from cache_path (eg vendor/cache) ?
I'm trying to switch from Dependabot, and missing their `vendor: true` option.

**Comment by rarkins** [comment, marked as answer]
Somehow we never got asked for this. Can you write to a feature request with an example public repo which would need vendor updates? Should be fairly fast to implement.

**Reply by jdelStrother** [reply]
OK, filed as https://github.com/renovatebot/renovate/issues/11004
